c++/src/capnp/reconnect.h

```diff
@@ -21,7 +22,7 @@
 
 #pragma once
 
-#include "capability.h"
+#include <capnp/capability.h>
```

Comment:
Isn't this in the same library?

Reply:
It is `capnp-rpc`, not `capnp`.

Reply:
Oh I see, capability.h is in the base library while capability.c++ is in the RPC library, due to the weird compile-time vs. link-time dependency differences.